models.py

```diff
@@ -53,6 +53,33 @@ def serialize(self):
         }
 
 
+class Chapter(db.Model):
+    __tablename__ = 'ChapterData'
+
+    def serialize(self):
+        return {
+            'chapterNumber': str(self.babID),
+            'chapter': [
+                {
+                    'lang': 'ar',
+                    'bookId': str(self.arabicBookID),
```

Comment:
We're getting rid of the `bookID` field. Let's use the same map as for `Book` here applied to `self.arabicBookID` and move it outside the language-specific block.

Reply:
And call it `bookNumber`, of course.